test/Microsoft.ML.Functional.Tests/Common.cs

```diff
@@ -20,5 +20,34 @@ public static void CheckMetrics(RegressionMetrics metrics)
             Assert.True(metrics.L2 >= 0);
             Assert.True(metrics.RSquared <= 1);
         }
+
+        public static void AssertEqual(float[] array1, float[] array2)
+        {
+            Assert.NotNull(array1);
+            Assert.NotNull(array2);
+            Assert.Equal(array1.Length, array2.Length);
+
+            for (int i = 0; i < array1.Length; i++)
+                Assert.Equal(array1[i], array2[i]);
+        }
+
+        public static void AssertEqual(Schema schema1, Schema schema2)
```

Comment:
>Schema [](start = 39, length = 6)

Schould Schema implement IEquitable<Schema> ? 
[https://stackoverflow.com/questions/8400028/comparing-two-instances-of-a-class](https://stackoverflow.com/questions/8400028/comparing-two-instances-of-a-class)

Reply:
That would be awesome.

---
In reply to: [256209052](https://github.com/dotnet/machinelearning/pull/2518#discussion_r256209052) [](ancestors = 256209052)

Reply:
+1 on implementing IEquitable. Should have an issue about it. 

---
In reply to: [256209052](https://github.com/dotnet/machinelearning/pull/2518#discussion_r256209052) [](ancestors = 256209052)